steamcmd_servers/insurgency_sandstorm/egg-insurgency--sandstorm.json

```diff
@@ -1,14 +1,19 @@
 {
     "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
     "meta": {
-        "version": "PTDL_v1"
+        "version": "PTDL_v1",
+        "update_url": null
     },
-    "exported_at": "2020-06-27T05:17:21-04:00",
+    "exported_at": "2021-02-16T10:20:13+11:00",
     "name": "Insurgency: Sandstorm",
-    "author": "brycea@terrahost.cloud",
+    "author": "playstrikemc@outlook.com",
```

Comment:
Why was this changed?

Reply:
Hi Parker,
I believe playstrikemc was the original author of the egg. I made some tweaks to a copy that another user had already tweaked and they must have updated the email address :)

just setting it back to the original author

Reply:
bryce is the only one with their name on this file and it hasn't changed for 8 months since they uploaded it. Please change it back.